### ISSUE TITLE
Transition name mislead

### ISSUE THREAD

**Issue by mbarrben** [antoniolg/MaterialEverywhere#13]
It's confusing that you use `EXTRA_IMAGE` as an intent extra (https://github.com/antoniolg/MaterialEverywhere/blob/master/app/src/main/java/com/antonioleiva/materialeverywhere/DetailActivity.java#L74), 
but also as a transition name between views (https://github.com/antoniolg/MaterialEverywhere/blob/master/app/src/main/java/com/antonioleiva/materialeverywhere/DetailActivity.java#L72). 
More confusing is that the `ImageView` in `DetailActivity` has a transition name defined (https://github.com/antoniolg/MaterialEverywhere/blob/master/app/src/main/res/layout/activity_detail.xml#L32), 
but later overridden with `EXTRA_IMAGE` (https://github.com/antoniolg/MaterialEverywhere/blob/master/app/src/main/java/com/antonioleiva/materialeverywhere/DetailActivity.java#L41).

It would be clearer if the transition name defined in the layout is the same that is used in `ActivityOptionsCompat.makeSceneTransitionAnimation` call. Then there's no need to call `ViewCompat.setTransitionName`. Even better, that transition name can be a string resource.

**Comment by zipcerio** [comment]
@mbarrben I agree with you. At first when I was looking at the source, it took me awhile to understand it until I checked other sources.

**Comment by sharathyadhav1** [comment]
error: Error: No resource found that matches the given name: attr 
 'android:windowAllowReturnTransitionOverlap'.
